src/Render/FriendicaSmarty.php

```diff
@@ -64,7 +65,7 @@ public function __construct(string $theme, array $theme_info, string $work_dir)
 		 * RAM available + have enabled caching inode tables (aka.
 		 * "descriptors"). Still it won't hurt you.
 		 */
-		$this->setUseSubDirs(true);
+		$this->setUseSubDirs(DI::config()->get('smarty3', 'use_sub_dirs'));
```

Comment:
Please add a new parameter `bool $useSubDirs` at the constructor and pass it through the call of it